### PR TITLE
Steve Irwin Soumya upwork

### PR DESCRIPTION
**Pull request by soummyaah**

### CODE REVIEW

**Comment by BaseMax** [comment]
LGTM

File: tests/spelling/misspelled-words-temp.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+./doc/source/colorado.md
```
Comment (jbampton) [inline tests/spelling/misspelled-words-temp.txt:1]: What is this ?

**Review by jbampton — CHANGES_REQUESTED** [review]
Why are you removing this file ?

`tests/spelling/misspelled-words.txt`

File: tests/spelling/misspelled-words-temp.txt (hunk 1 above)
Comment (soummyaah) [reply tests/spelling/misspelled-words-temp.txt]: My bad, reverted those changes.

File: doc/source/index.rst [hunk 2]
```diff
@@ -36,6 +36,7 @@ Welcome to slurp's documentation!
    teresa.md
    al-latrun.md
    wasfi.md
+   steve-irwin.md
```
Comment (jbampton) [inline doc/source/index.rst:39]: The menu needs to be in alphabetical order by document title.

https://slurp--1093.org.readthedocs.build/en/1093/

Screen Shot 2021-07-26 at 6 30 49 pm

File: doc/source/steve-irwin.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+His wildlife warrior’s charity bought up hundreds of square miles around the world for wildlife conservation, the charity which is still operating today is also involved in conservation efforts for animals like Sumatran tigers, koalas, Cambodian elephants, and more.  
+
+The Australia Zoo even manages a 500 plus square mile reserve in the north of Queensland named after her win whether or not you approved of us has up close an aggressive approach to videotaping animals it's clear pop culture could use more science and wildlife celebrities 
+
+Like urban for too many. the enormous diversity of life on Earth as a remote abstraction something that belongs to another more fantastical world people like urban Jane Goodall and David Attenborough help us appreciate the natural world by bringing it into our homes and selling us on its importance and vulnerability with their care, Wonder, thoughtfulness, and, enthusiasm.  
+
+## About the Author:
```
Comment (jbampton) [inline doc/source/steve-irwin.md:25]:
```suggestion
## About the Author

```

**Review by jbampton — CHANGES_REQUESTED** [review]

File: doc/source/index.rst (hunk 2 above)
Comment (soummyaah) [reply doc/source/index.rst:39]: Pushed the changes.

**Review by jbampton — CHANGES_REQUESTED** [review]
Alphabetical order by document title not by filename.

Don't change stuff I don't ask.

Like the "60s" was an extra change not needed.

https://slurp--1093.org.readthedocs.build/en/1093/

Screen Shot 2021-07-26 at 8 16 05 pm

**Comment by soummyaah** [comment]
@jbampton Reverted and changed only for the page I added.

**Review by jbampton — APPROVED** [review]

**Pull request merged**